app/screens/error_teams_list/error_teams_list.test.js

```diff
@@ -32,32 +26,31 @@ describe('ErrorTeamsList', () => {
         const wrapper = shallow(
             <ErrorTeamsList {...baseProps}/>
         );
+
+        expect(wrapper.find(FailedNetworkAction).exists()).toEqual(true);
         expect(wrapper.getElement()).toMatchSnapshot();
     });
 
-    test('should call for userInfo on retry', async () => {
-        const connection = jest.fn();
-        const selectDefaultTeam = jest.fn();
-        const logout = jest.fn();
-        const actions = {
-            ...baseProps.actions,
-            loadMe,
-            logout,
-            selectDefaultTeam,
-            connection,
-        };
-
-        const newProps = {
-            ...baseProps,
-            actions,
-        };
-
+    test('should call actions on getUserInfo', async () => {
         const wrapper = shallow(
-            <ErrorTeamsList {...newProps}/>
+            <ErrorTeamsList {...baseProps}/>
         );
 
-        wrapper.find(FailedNetworkAction).props().onRetry();
-        await loadMe();
-        expect(selectDefaultTeam).toHaveBeenCalledTimes(1);
+        const instance = wrapper.instance();
+        await instance.getUserInfo();
+
+        expect(baseProps.actions.connection).toHaveBeenCalledTimes(1);
+        expect(baseProps.actions.connection).toBeCalledWith(true);
+
+        expect(baseProps.actions.loadMe).toHaveBeenCalledTimes(1);
+        expect(baseProps.actions.loadMe).toBeCalledWith();
+
+        expect(baseProps.actions.resetToChannel).toHaveBeenCalledTimes(1);
+        expect(baseProps.actions.resetToChannel).toBeCalledWith({disableTermsModal: true});
+
+        expect(baseProps.actions.selectDefaultTeam).toHaveBeenCalledTimes(1);
+        expect(baseProps.actions.selectDefaultTeam).toBeCalledWith();
+
+        expect(wrapper.state('loading')).toEqual(false);
```

Comment:
`onPress` cannot be called on this component.  It can be tested in FormattedText component but since it's also passed into react-native's Text component, it don't make sense to test since it's already covered by the library itself.  Added unit test is to check functionality of getUserInfo function.